### ISSUE TITLE
Hair is flying arbitrarily around

### ISSUE THREAD

**Issue by SavAct** [noname0310/babylon-mmd#20]
The hairs of many models fly around wildly, even if no animation file is connected. This is also the case with some accessories. As if it had no mass and the slightest fluctuations cause it to whirl around randomly.
![2024-04-25](https://github.com/noname0310/babylon-mmd/assets/43270815/747cbd69-5468-4ff4-9e69-78e8297ddcab)

**Comment by noname0310** [comment]
try
```typescript
const mmdPhysics = new MmdPhysics(scene);
mmdPhysics.angularLimitClampThreshold = 10 * Math.PI / 180; // 10 or higher

const mmdRuntime = new MmdRuntime(scene, mmdPhysics);
```
This is an issue with havok physics, so it's not a fundamental fix.

If this doesn't work, you should consider bake the physics simulation.

I am currently working on bullet physics(the physics engine of mmd). Once this is finalized, there will be no problems with physics computation, but it will take a long time before this solution is available.

So for now, there is no solution other than the two methods suggested above. 
If you're interested in physics baking, I'll show you how. in detail

**Comment by noname0310** [comment]
I can easily reproduce mmd's physics using ammo.js, the emscripten build of bullet physics, but I'm not currently working on an ammo.js implementation as I want to focus on a more optimized solution.

Creating an ammo version of `MmdPhysics` would be an trivial task, so if you're interested, you might want to consider it.

**Comment by SavAct** [comment]
Thanks for the information. I am curious about your optimized solution 🙌

**Comment by SavAct** [comment]
Thanks for the information. I am curious about your optimized solution 🙌

**Comment by noname0310** [comment]
I'm going to handle the bullet physics in a way that multithreads it and makes the JS interrupt overhead nearly non-existent.

It's currently in progress in a separate repository.
https://github.com/noname0310/bullet-rust-wasm

**Comment by noname0310** [comment]
Since I received several similar requests, I decided to quickly create a non-performant implementation first

The ammo.js physics backend will be available from babylon-mmd 0.42.0.
Although ammo.js is not very good performance, you will get very similar results to mmd.

The usage is the same as before, see this test code
https://github.com/noname0310/babylon-mmd/blob/main/src/Test/Scene/physicsTestScene.ts#L63
https://github.com/noname0310/babylon-mmd/blob/main/src/Test/Scene/physicsTestScene.ts#L111-L113
